metricbeat/module/apache/status/status.go

```diff
@@ -12,6 +12,8 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/apache"
 )
 
+const AUTO_STRING  = "?auto"
```

Comment:
Making this a variable instead of a constant would make it possible to overwrite it during testing (if needed).